Tasks/Kuzhovnik/Task1/Task1.py

```diff
@@ -0,0 +1,82 @@
+import math
+from decimal import *
+
+while True:
+    p = input("Enter deposit amount:\n")
+    try:
+        if Decimal(p) <= 0:
+            raise Exception
+        Decimal(p)
+        break
+    except Exception:
+        print("Incorrect value!")
+p = Decimal(p)
+
+while True:
+    t = input("Enter deposit period, years:\n")
+    try:
+        if Decimal(t) <= 0:
+            raise Exception
+        Decimal(t)
+        break
+    except Exception:
+        print("Incorrect value!")
+t = Decimal(t)
+
+while True:
+    r = input("Enter deposit rate, pct.:\n")
+    try:
+        if Decimal(r) <= 0:
+            raise Exception
+        Decimal(r)
+        break
+    except Exception:
+        print("Incorrect value!")
+r = Decimal(r)
+
+while True:
+    cap = str(input("Use month capitalisation? Write 'Yes' or 'No'\n"))
+    try:
+        if cap.lower() == "yes":
+            n = Decimal(12)
+        elif cap.lower() == "no":
+            n = Decimal(1)
+        else:
+            raise Exception
+        break
+    except Exception:
+        print("Incorrect value!")
```

Comment:
the validation approach is good enough, but it would be better to check all parameters in a single cycle, we will discuss this approach in details as soon as everybody are familiar with the syntax